File: mobile/src/main/java/ro/antiprotv/sugar/MainActivity.java
package ro.antiprotv.sugar;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentPagerAdapter;
import androidx.viewpager.widget.ViewPager;

import com.google.android.material.tabs.TabLayout;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        FragmentPagerAdapter fpa = new MainPagerAdapter(this, getSupportFragmentManager());
        ViewPager viewPager = findViewById(R.id.view_pager);
        viewPager.setAdapter(fpa);
        TabLayout tabs = findViewById(R.id.main_layout_tabs);
        tabs.setupWithViewPager(viewPager);
    }

}

File: mobile/src/main/java/ro/antiprotv/sugar/MainActivity.kt
package ro.antiprotv.sugar

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*
import ro.antiprotv.sugar.ui.MainPagerAdapter

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        view_pager.adapter = MainPagerAdapter(this, supportFragmentManager)
        main_layout_tabs.setupWithViewPager(view_pager)
    }
}